### PR TITLE
Fix various typos in the templates (Ruby, C#)

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

As discussed with @katrinleinweber, here is her commit to fix the typos.

### CODE REVIEW

File: samples/client/petstore/csharp-dotnet2/OpenAPIClientTest/Lib/OpenAPIClient/README.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ namespace Example
             Configuration.Default.AccessToken = "YOUR_ACCESS_TOKEN";
 
             var apiInstance = new PetApi();
-            var pet = new Pet(); // Pet | Pet object that needs to be added to the store
+            var body = new Pet(); // Pet | Pet object that needs to be added to the store
```
Comment (katrinleinweber) [inline samples/client/petstore/csharp-dotnet2/OpenAPIClientTest/Lib/OpenAPIClient/README.md]: I'm don't know about this `body` (and many more below). @wing328: I'm guessing they are your additional renames squashed into the cherry-picked typo corrections

Comment (wing328) [reply samples/client/petstore/csharp-dotnet2/OpenAPIClientTest/Lib/OpenAPIClient/README.md]: Good catch. Those changes are not related to your commit. It's due to an enhancement we made to retain the body parameter name defined in OpenAPI v2 spec.

**Comment by katrinleinweber** [comment]
I found another one: https://github.com/katrinleinweber/openapi-generator/commit/22ea5946fcc25c535285d044e7d3e7fa811ecf86. Cherry-pick again, or separate PR?

**Comment by wing328** [comment]
@wing328 I'll make the same change manually in this PR if that's ok with you.

**Pull request merged**